### PR TITLE
Improving documentation about tools and versions required/minimum

### PR DESCRIPTION
**Pull request by MV88**
## Description

Improving documentation  about tools and versions required/minimum

**NOTE:** I have also changed the link in the readme to open in a new tab

RtD [demo link](https://ms2-requirements.readthedocs.io/en/4959_requirements/developer-guide/requirements/)

image

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [x] Other... Please describe: documentation

## Issue

**What is the current behavior?**

#4959

**What is the new behavior?**

doc improved and centralized

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Comment by tdipisa** [comment]
@MV88 please provide a build of your doc branch in RdD to help the review

**Comment by offtherailz** [comment]
I suggest only to place the Installation section before debug/build

**Comment by MV88** [comment]
> @MV88 please provide a build of your doc branch in RdD to help the review

you mean that the screenshot is not enough ?

**Comment by MV88** [comment]
@tdipisa i've added the readthedocs link into the description

File: README.md [hunk 1]
```diff
@@ -6,67 +6,71 @@
...
 License
 ------------
-MapStore is Free and Open Source software, it is based on OpenLayers, Leaflet and [ReactJS](https://facebook.github.io/react/), and is licensed under the Simplified BSD License.
+MapStore is Free and Open Source software, it is based on OpenLayers, Leaflet and <a href="https://facebook.github.io/react/" target="_blank">ReactJS</a>, and is licensed under the Simplified BSD License.
 
 
 Demo Instances
 ---------------
 We have the following instances:
+<a href="" target="_blank"></a>
```
Comment (mbarto) [inline README.md:69]: Typo?

**Comment by tdipisa** [comment]
@offtherailz @mbarto, @mbarto has provided the required changes, can we merge this PR?

**Pull request merged**